pkg/volume/host_path/host_path.go

```diff
@@ -198,3 +213,39 @@ func (r *hostPathRecycler) Recycle() error {
 	}
 	return volume.RecycleVolumeByWatchingPodUntilCompletion(pod, r.host.GetKubeClient())
 }
+
+// hostPathCreater implements a dynamic provisioning Creater for the HostPath plugin
+// This implementation is meant for testing only and only works in a single node cluster.
+type hostPathCreater struct {
+	host    volume.VolumeHost
+	options volume.VolumeOptions
+}
+
+// Create for hostPath simply creates a local /tmp/hostpath_pv/%s directory as a new PersistentVolume.
+// This Creater is meant for development and testing only and WILL NOT WORK in a multi-node cluster.
+func (r *hostPathCreater) Create() (*api.PersistentVolume, error) {
+	fullpath := fmt.Sprintf("/tmp/hostpath_pv/%s", util.NewUUID())
```

Comment:
This isn't going to work with SELinux enforcing.  The SELinux type will be `tmp_t` which is (rightfully) not usable from the SELinux context docker containers get.